perfkitbenchmarker/pkb.py

```diff
@@ -147,6 +147,14 @@
                      'all disks together.',
                      lower_bound=1)
 
+# support for proxy
+flags.DEFINE_string('http_proxy', '',
+                    'Specify a proxy for HTTP in the form [user:passwd@]proxy.server:port.')
```

Comment:
There's some ambiguity about where the proxies get used. Maybe clarify that these are for use within the cloud deployed OS? Presumably if people need proxies on the machine where they are running pkb.py, those would be configured separately.